app/knights.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import annotations
+from typing import Union
+
+
+class Knights:
+
+    def __init__(
+            self, name: str,
+            power: int, hp: int,
+    ) -> None:
```

Comment:
Тут ще відступи краще пофіксити
```suggestion
    def __init__(
            self, 
            name: str,
            power: int, 
            hp: int,
    ) -> None:
```

Reply:
Як правило, ми ставимо все в один рядок, або кожному по рядку виділяємо
Це щоб, якщо будуть мінятися вхідні дані, можна було швидко відредагувати, або додати новий арг